### DISCUSSION TITLE
Please ad Cohere's new embedding model.

### DISCUSSION THREAD

**Discussion by vdruts** [brianpetro/obsidian-smart-connections#850] [General]
https://cohere.com/blog/multimodal-embed-3?utm_medium=email&_hsenc=p2ANqtz-9iqTHVHPV_Eusc1Iqa791xpN1w1KWQ6QhMa7EAPL7stWBFzTqdcTz7xfKSm28eQmUGHs5hAtGjIPpBPa9-oUQQZ5fzmA&_hsmi=330505833&utm_content=330505833&utm_source=hs_email

The question is, how can we make use of it's image capabilities through Obsidian/SC

**Comment by brianpetro** [comment]
Cool 🌴
